test/archive.cpp

```diff
@@ -297,7 +297,7 @@ TEST(ZimArchive, illustration)
 }
 
 struct ZimFileInfo {
-  zim::entry_index_type articleCount, entryCount, allEntryCount;
+  zim::entry_index_type articleCount, entryCount, allEntryCount, mediaCount;
```

Comment:
Have you considered adding `mediaCount` to `TestDataInfo`?

Reply:
No :)
But considering it, I agree that it would be added to `TestDataInfo` instead of `ZimFileInfo` as the value is the same whatever we are with namespace or not.
But all the `*Count` are in `ZimFileInfo`, I think it would be more coherent to keep them there. If not, we should also move 
`articleCount` in `TestDataInfo` as it is now the same value also.

Reply:
> But considering it, I agree that it would be added to `TestDataInfo` instead of `ZimFileInfo` as the value is the same whatever we are with namespace or not.

Then keeping `mediaCount` in `ZimFileInfo` is not justified.

> But all the `*Count` are in `ZimFileInfo`, I think it would be more coherent to keep them there. If not, we should also
> move `articleCount` in `TestDataInfo` as it is now the same value also.

`articleCount` can/should be moved out of `ZimFileInfo` only if it becomes independent of the namespace scheme by design, not if it just happens to match in these few cases.

Reply:
> Then keeping mediaCount in ZimFileInfo is not justified.

Done in a fixup commit